common/lib/xmodule/xmodule/util/adaptive_learning.py

```diff
@@ -211,8 +219,12 @@ def get_knowledge_node_students(self, student_uid):
         """
         url = self.knowledge_node_students_url
         payload = {'student_id': student_uid, 'key_type': 'uid'}
-        response = requests.get(url, headers=self.request_headers, data=payload)
-        links = json.loads(response.content)
+        try:
+            links = []
```

Comment:
@macornwell This should go into the [`except` block](https://github.com/open-craft/edx-platform/pull/69#discussion_r148497812).